### ISSUE TITLE
Some components are not accessible by screen reader

### ISSUE THREAD

**Issue by patriciasaramela** [ant-design/ant-design#13198]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Some visually impaired users tested the Antd documentation components and were unable to use some because they are not accessible using a screen reader.
Here is the list of some non-accessible components:

- Pagination
- Autocomplete
- Checkbox (works only on firefox)
- Rate
- Select
- Tree
- Avatar
- Badge
- Calendar
- Popover
- Tooltip
- Drawer
- Message
- Notification
- Progress
- Popconfirm

### What does the proposed API look like?
-

**Comment by IssuehuntBot** [comment]
@issuehunt has cancelled funding for this issue.(Cancelled amount: $20.00) [See it on IssueHunt](https://issuehunt.io/repos/34526884/issues/13198)

**Comment by afc163** [comment]
trace #22343
